src/Core/Domain/Product/ProductTaxRulesGroupSettings.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\Product;
+
+/**
+ * Some static settings of tax rules group inside Product context
+ */
+class ProductTaxRulesGroupSettings
+{
+    /**
+     * Value of tax rules group which reflects that product has no tax rules group applied
+     * Null value doesn't fit because it is used to identify partial updates
+     */
+    const NONE_APPLIED = 0;
```

Comment:
not sure about this one. 
there was similar discussion here https://github.com/PrestaShop/PrestaShop/pull/15611/files#r327987588

Reply:
it works for me